src/main/java/duke/command/DeletePatientTaskCommand.java

```diff
@@ -66,31 +67,33 @@ public void execute(PatientTaskList patientTaskList, TaskManager tasks, PatientM
         if (patientId != 0) {
             try {
                 Patient toBeDeletedPatient = patientManager.getPatient(patientId);
-                for (PatientTask patientTask: patientTaskList.getPatientTask(patientId)) {
-                    if (patientTask.getUid() == taskId) {
-                        patientTaskList.deletePatientTaskByUniqueId(taskId);
-                        patientTaskStorage.save(patientTaskList.fullPatientTaskList());
-                        ui.patientTaskDeleted(patientTask, toBeDeletedPatient);
-                    }
+                if (patientTaskList.isPatientIdExist(patientId)) {
```

Comment:
Small grammar issue, but the method should be refactored from `.isPatientIdExist()` to `.doesPatientIdExist()`.

Reply:
noted